### DISCUSSION TITLE
Achievements?

### DISCUSSION THREAD

**Discussion by matheusrcruz** [community/community#48078] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

What is the purpose of Achievements?

**Comment by CodeMaster7000** [comment, marked as answer]
Achievements can give the user who gets them satisfaction, but they are also useful for others. For example, if someone has a "starstruck" achievement, it will bring attention to the most renowned projects a person has created. It can also indicate your commitment and contribution to the GitHub community, as someone with a "galaxy brain" achievement shows that he/she is willing to help out others. In a nutshell, they are tokens of appreciation for your hard work on GitHub.

**Reply by matheusrcruz** [reply]
Thanks you, I understand.

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Hazrat-Ali9** [comment]
How do I get a Galaxy Brain badge
Response
